### ISSUE TITLE
[Attention] OpenMMLab Codecamp

### ISSUE THREAD

**Issue by hhaAndroid** [open-mmlab/mmyolo#302]
![image](https://user-images.githubusercontent.com/17425982/202830552-7b1c684c-7539-42fb-8dee-3ed8b1e3e7e3.png)

# Introduction
Interested in deeply participating in OpenMMLab projects? Want to learn more about OpenMMLab's awesome tools without wasting plenty of time reading docs? The First OpenMMLab Codecamp has begun!! We provide more than a hundred tasks from seventeenth research directions for you to pick. Whether you are a novice in AI or a senior developer, there are suitable tasks for you to participate in. We will provide quick response and full guidance to help you smoothly complete those tasks and upgrade to a core contributor of OpenMMLab. We combined Beijing Super Cloud Center to support with computing power. 

How to participate? 

Select the task you are interested in and submit registration [here](https://openmmlab.com/activity/codecamp/apply). We will inform you in three days if you have enrolled for the tasks, and then you can formulate the task plan with tutor and start development ! Once your PR has passed preliminary review, you can apply for the next task or just wait for the award!
More details: [OpenMMLab Activity page](https://openmmlab.com/activity/codecamp)。

# Tasks
Here, we listed the tasks in MMYOLO, welcome to submit registration!

<meta charset="utf-8"><div data-page-id="Cifedm5GBovxjixU4Vdcd7mwndj" data-docx-has-block-data="true"><div>

任务题目Task | 任务描述Discription | 技术标签Related skills | 难易程度Difficulty | 积分数Credits
-- | -- | -- | -- | --
Algorithm  YOLOv7 | Implement YOLOv7 algorithm.- Official weight conversion script based on the proposed PR in MMYOLO- Implementing the P6 large model loss logic based on the proposed PR in MMYOLO | Python, PyTorch, Object Detection | hard | 80
YOLOv5 Support ignore instance | Support ignore instance reference MMDet related logic in YOLOv5 | Python, PyTorch, Object Detection | hard | 80
YOLOv5 supports the data transformation part of the instance segmentation task | The data transformation part of the instance segmentation task is supported in MMYOLO by referring to the official YOLOv5 instance segmentation | Python, PyTorch, Instance Segmentation | hard | 80
YOLOv5 supports the loss part of the instance segmentation task | The loss part of the instance segmentation task is supported in MMYOLO by referring to the official YOLOv5 instance segmentation | Python, PyTorch, Instance Segmentation | hard | 80
Support for PPYLOE deployments | Support for PPYLOE deployments based on MMDeploy | Python, PyTorch, Deployment | hard | 80
Automatic anchor optimization support in YOLOv5 training | Supports automatic anchor optimization in YOLOv5 training based on optimized anchor scripts | Python, PyTorch, Object Detection | medium | 30
Provide examples of the combination of algorithm modules in MMYOLO | Provide examples documentation on the combination of algorithm modules for the various YOLO algorithms implemented in MMYOLO | Documentation | medium | 30

Sign Up Here : [application form](https://openmmlab.com/activity/codecamp/apply) 

By the way, we strongly encourage you to publish your experience on social media like medium or twitter with tag "OpenMMLab Codecamp" to share your experience with more developers!
[https://discord.gg/KuWMWVbCcD](https://cdn.vansin.top/open-mmlab/cjsky.jpg)

Discussion group: discord link

Welcome to join the discussion below or in discord. Come to take the challenge, and become a contributor to the OpenMMLab !
